pkg/network/setup/podnic.go

```diff
@@ -91,6 +92,15 @@ func newPhase2PodNIC(vmi *v1.VirtualMachineInstance, network *v1.Network, handle
 	if err != nil {
 		return nil, err
 	}
+	if podnic.vmiSpecIface.SRIOV != nil {
+		return podnic, nil
```

Comment:
This is changing the current logic (the other operations are not done now).
While this may be ok, it is a side effect.

Do we have other options at this? Maybe at the usage?

Reply:
We can change it to return `nil`, and change the caller to process non-nil objects.

Alternatively, we can filter non-SRIOV networks at the caller (so no podNIC object is created for SR-IOV networks).

I like the latter.

Reply:
I changed it to filter non-SRIOV networks at the caller.